mne/preprocessing/ica.py

```diff
@@ -120,7 +120,23 @@ class ICA(ContainsMixin):
     u"""M/EEG signal decomposition using Independent Component Analysis (ICA).
 
     This object can be used to estimate ICA components and then remove some
-    from Raw or Epochs for data exploration or artifact correction.
+    from Raw, Epochs or Evoked for data exploration or artifact correction.
+    
+    Prior to fitting and applying the ICA, data is whitened (de-correlated and
+    scaled to unit variance, also called sphering transformation) by means of
+    a Principle Component Analysis (PCA). In addition to the whitening, this step
+    introduces the option to reduce the dimensionality of the data, both, prior to
+    fitting the ICA and prior to reverting to sensor space, controllable by the
+    two parameters `max_pca_components` and `n_pca_components`, respectively.
+    .. note:: Commonly used for reasons of i) computational efficiency and
+              ii) additional noise reduction, it is a matter of current debate
+              whether pre-ICA dimensionality reduction could decrease the reliability
+              and stability of the ICA, at least for EEG data and especially during
+              preprocessing [5].
+              On the other hand, for rank-deficient data such as EEG data after 
+              average reference, it is recommended to reduce the dimensionality by 1
```

Comment:
Maybe something like:
```suggestion
              average reference or interpolation, it is recommended to reduce
              the dimensionality (by 1 for average reference and 1 for each
              interpolated channel) for optimal ICA performance [6].
```